src/XIVLauncher.Core/CoreEnvironmentSettings.cs

```diff
@@ -16,6 +16,7 @@ public static class CoreEnvironmentSettings
     public static bool ClearAll => CheckEnvBool("XL_CLEAR_ALL");
     public static bool? UseSteam => CheckEnvBoolOrNull("XL_USE_STEAM"); // Fix for Steam Deck users who lock themselves out
     public static bool IsSteamCompatTool => CheckEnvBool("XL_SCT");
+    public static int AltAppID => GetAltAppId(System.Environment.GetEnvironmentVariable("XL_APPID"));
```

Comment:
Usually nullability is used here to indicate an env var not existing, instead of using a special value (like -1).